modules/chassis-aws-analytics/manifests/init.pp

```diff
@@ -2,42 +2,21 @@
 	$config
 ) {
 
-	# Install Docker
-	include 'docker'
-
-	# Install Docker Compose
-	class { 'docker::compose':
-		ensure => present,
-	}
-
-	if ( ! empty( $config[disabled_extensions] ) and 'humanmade/chassis-aws-analytics' in $config[disabled_extensions] ) {
-		$dir = absent
-		$compose_file = absent
+	# Allow disabling the extension
+	if ( !empty($config[disabled_extensions]) and 'humanmade/chassis-aws-analytics' in $config[disabled_extensions] ) {
+		$status = stopped
+		$active = false
 	} else {
-		$dir = directory
-		$compose_file = present
+		$status = running
+		$active = true
 	}
 
-	# Add the docker-compose.yaml file.
-	file { '/home/vagrant/aws-analytics':
-		ensure => $dir,
-	}
-	file { '/home/vagrant/aws-analytics/docker-compose.yaml':
-		ensure  => $compose_file,
-		mode    => '0777',
-		content => template('chassis-aws-analytics/docker-compose.yaml'),
-		require => [
-			File['/home/vagrant/aws-analytics'],
-		],
-	}
+	$services = [ 'cognito', 'pinpoint' ]
 
-	# Start the stack.
-	docker_compose { '/home/vagrant/aws-analytics/docker-compose.yaml':
-		ensure  => $compose_file,
-		require => [
-			Class['docker::compose'],
-			File['/home/vagrant/aws-analytics/docker-compose.yaml'],
-		],
+	# Create services.
+	analytics_service { $services:
```

Comment:
You could move it inline if you wanted, but yolo

Reply:
Get that carbon footprint down a fraction 😛